src/app/admin/page.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from "react";
+import Title from "@/components/Title/Title";
+import { Metadata } from "next";
+import AdminPage from "@/app/admin/AdminPage";
+import { getUsers } from "@/app/admin/adminActions";
+
+// disables caching
+export const revalidate = 0;
```

Comment:
How come we need to disable caching in this case? Not suggesting it isn't needed, just don't fully understand why we do this.

Reply:
Emm, I believe we were having some problems with results from calls to the edge functions caching things that shouldn't be cached - I'll have more of a look into this later, it's on my `TODO` list